spec/requests/properties_controller_spec.rb

```diff
@@ -49,7 +49,7 @@
         end
       end
 
-      context "Invalid assessment ID causes service to return failure" do
+      context "with invalid assessment ID" do
         let(:assessment_id) { SecureRandom.uuid }
 
         it "returns expected error response", :show_in_doc do
```

Comment:
:show_in_doc is an api pie helper i believe

Reply:
removed